samples/MvcApp/MvcApp.csproj

```diff
@@ -283,7 +283,7 @@
           <AutoAssignPort>True</AutoAssignPort>
           <DevelopmentServerPort>50568</DevelopmentServerPort>
           <DevelopmentServerVPath>/</DevelopmentServerVPath>
-          <IISUrl>https://localhost:44339/</IISUrl>
+          <IISUrl>http://localhost:44339/</IISUrl>
```

Comment:
???

Reply:
oops - didn't mean to commit that. That was part of verifying the behavior between ssl/non-ssl. I'll revert it